apps/examples/testcase/ta_tc/ble_manager/utc/utc_blemanager_main.c

```diff
@@ -0,0 +1,1172 @@
+	TC_ASSERT_EQ("ble_manager_init", ble_manager_init(&server_config), BLE_MANAGER_SUCCESS);//server role
+	TC_ASSERT_EQ("ble_manager deinit", ble_manager_deinit(), BLE_MANAGER_SUCCESS);
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_blemanager_init_n(void)
+{
+	TC_ASSERT_EQ("ble_manager_init", ble_manager_init(&server_config), BLE_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("ble_manager_init", ble_manager_init(&server_config), BLE_MANAGER_ALREADY_WORKING);
+	TC_ASSERT_EQ("ble_manager deinit", ble_manager_deinit(), BLE_MANAGER_SUCCESS);
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_blemanager_deinit_p(void)
+{
+	TC_ASSERT_EQ("ble_manager_init", ble_manager_init(NULL), BLE_MANAGER_SUCCESS);//not server role
+	TC_ASSERT_EQ("ble_manager deinit", ble_manager_deinit(), BLE_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("ble_manager_init", ble_manager_init(&server_config), BLE_MANAGER_SUCCESS);//server role
+	TC_ASSERT_EQ("ble_manager deinit", ble_manager_deinit(), BLE_MANAGER_SUCCESS);
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_blemanager_deinit_n(void)
+{
+	TC_ASSERT_EQ("ble_manager deinit", ble_manager_deinit(), BLE_MANAGER_INVALID_STATE);
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_blemanager_mac_p(void)
+{
+	TC_ASSERT_EQ("ble_manager_init", ble_manager_init(NULL), BLE_MANAGER_SUCCESS);
+	uint8_t mac[BLE_BD_ADDR_MAX_LEN];
+	TC_ASSERT_EQ("ble_manager_get_mac_addr", ble_manager_get_mac_addr(mac), BLE_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("ble_manager deinit", ble_manager_deinit(), BLE_MANAGER_SUCCESS);
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_blemanager_mac_n(void)
+{
+	TC_ASSERT_EQ("ble_manager_init", ble_manager_init(NULL), BLE_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("ble_manager_get_mac_addr", ble_manager_get_mac_addr(NULL), BLE_MANAGER_INVALID_ARGS);
+	TC_ASSERT_EQ("ble_manager deinit", ble_manager_deinit(), BLE_MANAGER_SUCCESS);
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_blemanager_get_bonded_device_p(void)
+{
+	TC_ASSERT_EQ("ble_manager_init", ble_manager_init(NULL), BLE_MANAGER_SUCCESS);
+	ble_bonded_device_list dev_list[BLE_MAX_BONDED_DEVICE] = { 0, };
+	uint16_t dev_count = 0;
+	ble_client_ctx *ctx;
+	ctx = ble_client_create_ctx(&client_config);
+	uint8_t dummy[] = {0x00, 0x1E, 0xC0, 0x04, 0x9F, 0xF3};
+	ble_conn_info conn_info = { 0, };
+	memcpy(conn_info.addr.mac, dummy, BLE_BD_ADDR_MAX_LEN);
+	conn_info.addr.type = BLE_ADV_TYPE_IND;
+	conn_info.conn_interval = 8;
+	conn_info.slave_latency = 128;
+	conn_info.mtu = 240;
+	conn_info.scan_timeout = 1000;
+	conn_info.is_secured_connect = true;
+	TC_ASSERT_EQ("ble_client_connect", ble_client_connect(ctx, &conn_info), BLE_MANAGER_SUCCESS);
+	CONTROL_VVDRIVER(VBLE_CMD_GEN_EVT, LWNL_EVT_BLE_CLIENT_CONNECT, 0, 3);
+	UTC_FUNC_WAIT;
+	TC_ASSERT_EQ("ble_client_connected cb", g_client_connected, BEVT_CONNECTED);
+	TC_ASSERT_EQ("ble_manager_get_bonded_device", ble_manager_get_bonded_device(dev_list, &dev_count), BLE_MANAGER_SUCCESS);
+	printf("Bonded device count: %d\n", dev_count);
```

Comment:
Is the `dev_count` needed to check?
If then, it is better to use `TC_ASSERT_EQ` macro.
If not, is this needed to print?

Reply:
Done